lib/init.coffee

```diff
@@ -1,25 +1,14 @@
 path = require 'path'
 
-{CompositeDisposable} = require 'atom'
+{BufferedNodeProcess} = require 'atom'
 
 module.exports =
-  config:
-    executablePath:
-      default: path.join __dirname, '..', 'node_modules', 'htmlhint', 'bin', 'htmlhint'
-      type: 'string'
-      description: 'HTMLHint Executable Path'
   activate: ->
-      console.log 'activate linter-htmlhint'
-      # console.log 'config', @config
-      # console.log 'dirname', __dirname
-      @subscriptions = new CompositeDisposable
-      @subscriptions.add atom.config.observe 'linter-htmlhint.executablePath',
-        (executablePath) =>
-          @executablePath = executablePath
-      @scopes =  ['text.html.angular', 'text.html.basic', 'text.html.erb', 'text.html.gohtml', 'text.html.jsp', 'text.html.mustache', 'text.html.handlebars', 'text.html.ruby']
+    @executablePath = path.resolve "#{__dirname}/../node_modules/htmlhint/bin/htmlhint"
```

Comment:
We need to retain the option to use a different htmlhint.  This is in use by several people.